config/manifests/bases/update-service-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |
```

Comment:
You should be able to [leave this blank][1] and have it inserted by `kustomize` magic at bundle-build time.

[1]: https://github.com/openshift/cincinnati-operator/pull/79/files#diff-d31bf97881e5efe5323e57c437e9999604ff7d312eef59930c415444b3e7c66aR5

Reply:
done